lib/bgs/service.rb

```diff
@@ -192,7 +196,7 @@ def get_ch33_dd_eft_info
     end
 
     def find_bank_name_by_routng_trnsit_nbr(routing_number)
-      return if routing_number.blank?
+      return if routing_number.blank? || routing_number == EMPTY_ROUTING_NUMBER
```

Comment:
I was thinking you could also do this
```suggestion
      return if routing_number.to_i.zero?
```

but the constant leaves an opportunity to separate business related logic and make a great supporting comment like you did.

Reply:
Not a bad idea, I'm gonna leave as-is for now but there might be an opportunity to go further with this check (figuring out the entire range of possibilities for the routing number and watching out for those)